sdk/highlight-node/src/client.ts

```diff
@@ -58,6 +58,7 @@ export class Highlight {
 
 		this.otel = new NodeSDK({
 			autoDetectResources: true,
+			resourceDetectors: [processDetectorSync],
```

Comment:
Should fix node processes for #6465. Will still need to validate other SDKs

Reply:
Here are the created services with the process data populated correctly.

<img width="1489" alt="Screenshot 2023-08-31 at 2 40 13 PM" src="https://github.com/highlight/highlight/assets/58678/f92b3abd-894d-4fdd-a0f3-cb667e901dc7">


Frontend parts aren't created yet per #6486